### PR TITLE
feat(settings): create settings page to manage user's preferences

### PR DESCRIPTION
**Pull request by rockingrohit9639**
- Create settings page which will manage user's preferences
   - Currency preference added 

# Screenshot
image

Fixes - #187 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205517400102816

### CODE REVIEW

**Comment by ivangarcilazo** [comment]
Hello :D! I hope you are well. I found the following problems, there is a loop that does not have the identifier and this can generate interface problems, also when I select for the first time the theme I want, a warning message pops up (second image), and notice that the monthly amount can be negative, although I do not know if that is in the plans
key-196

uncontrolled-196

negative-numbers-196

**Comment by ivangarcilazo** [comment]
Looks good :D

**Comment by rockingrohit9639** [comment]
> Hello :D! I hope you are well. I found the following problems, there is a loop that does not have the identifier and this can generate interface problems, also when I select for the first time the theme I want, a warning message pops up (second image), and notice that the monthly amount can be negative, although I do not know if that is in the plans key-196
> 
> uncontrolled-196
> 
> negative-numbers-196

Those inputs are not the part of settings page yet. These are just to make the page look good.
And, still API is needed to be integrated with the currency preference selector.

**Comment by mohitkhatri06** [comment]
Hey Rohit please do the changes for the charts also which are at "/"

**Comment by rockingrohit9639** [comment]
> Hey Rohit please do the changes for the charts also which are at "/"

I have pushed the changes.

**Review by mohitkhatri06 — APPROVED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by bharathkumarchowdary — APPROVED** [review]

**Comment by mohitkhatri06** [comment]
@ivangarcilazo  This one also.

**Review by ivangarcilazo — APPROVED** [review]

**Review by mohitkhatri06 — APPROVED** [review]

**Review by ivangarcilazo — APPROVED** [review]

File: src/pages/Settings/Settings.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <FormControl>
+          <InputLabel>Currency Unit</InputLabel>
+          <Select
+            value={preferences.currencyUnit}
+            name="currencyUnit"
+            label="Currency Unit"
+            onChange={handleValueChange}
+          >
+            {currencyCodes.map(({ name, symbol }) => (
+              <MenuItem value={symbol}>
```
[the remark this answers is not in the document]
Comment (mohitkhatri06) [reply src/pages/Settings/Settings.jsx:50]: I will create a GH issue for this lets merge this as for now.

**Review by bharathkumarchowdary — APPROVED** [review]

**Pull request merged**